src/main/ts/plugin.ts

```diff
@@ -51,6 +55,18 @@ export class ModuleFederationManifestPlugin {
     })
   }
 
+  static getHooks(compilation: webpack.Compilation): Hooks {
+    let hooks = compilationHooks.get(compilation)
+    if (!hooks) {
```

Comment:
Yes. But I have to get or set and return. In case of using `has` I need to cast to avoid `undefined` from `get` method or additional check

Reply:
I think `if (!hooks)` takes less LOC)

Reply:
Lets compare:
```js
if (!has(key)) {
  set(key, value)
}

return get(key)
```

Reply:
```typescript
  static getHooks(compilation: webpack.Compilation): Hooks {
    if (!compilationHooks.has(compilation)) {
      compilationHooks.set(compilation, {
        onManifestCreated: new AsyncSeriesHook(['manifest']),
      })
    }

    return compilationHooks.get(compilation)
  }
```

Raises ts error. Because `get` can return `undefined`
![image](https://user-images.githubusercontent.com/20929004/204314812-3591f889-2b1c-450d-a14f-d3487e2ebd99.png)


It can be fixed via 
```typescript
return compilationHooks.get(compilation)!
```

or

```typescript
return compilationHooks.get(compilation) as Hooks
```

But both cases look more counterintuitively

Reply:
The state of TS in 2022 (